templates/strapi4/files/.platform.app.yaml

```diff
@@ -0,0 +1,70 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+#
+# See https://docs.platform.sh/configuration/app.html
+
+# The name of this app. Must be unique within a project.
+name: app
+
+# The runtime the application uses. Strapi not yet suppported with NodeJS 14.
+type: nodejs:12
+
+# Use Yarn instead of npm.
+build:
+  flavor: none
+
+# Build dependencies.
+dependencies:
+  nodejs:
+    yarn: "1.22.5"
+
+# The hooks executed at various points in the lifecycle of the application.
+hooks:
+  build: |
+    # Download dependencies and build Strapi.
+    yarn
```

Comment:
```suggestion
    yarn --frozen-lockfile
```

Reply:
Shouldn't we change this? Doesn't _not_ using this flag modify the lock file on install?

Reply:
yes, i did exactly that yesterday, you removed the frozen lockfile tag

Reply:
I did?